8/jre/alpine/entrypoint.sh

```diff
@@ -28,11 +28,6 @@ TMPDIR=${TMPDIR:-/tmp}
 # JDK truststore location
 JRE_CACERTS_PATH=$JAVA_HOME/lib/security/cacerts
 
-# JDK8 puts its JRE in a subdirectory
```

Comment:
This change doesn't seem to mirror what other platforms are doing? e.g. the ubuntu noble one above.

Reply:
The 'ubuntu noble one' is a JDK rather than a JRE